main/main.js

```diff
@@ -1,15 +1,32 @@
 const electron = require('electron')
```

Comment:
This should also be part of the object destructuring

Reply:
# Hi PalmerAL :wave: 
exactly  yes, but i think, **may** that need to use **electron** module :wink: 

## **thanks for your review :heart:**

Reply:
Oh, nevermind, I misread what was happening here.